runtime/util/vmargs.c

```diff
@@ -221,6 +221,152 @@ findArgInVMArgs(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, UDATA match
 	return returnVal;
 }
 
+IDATA
+findArgInVMArgsImpl(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, UDATA match, const char* optionName, const char* optionValue, UDATA doConsumeArgs, UDATA doConsumeIgnoredArgs) {
```

Comment:
Changes look good, however instead of having two versions of this, what I meant was `findArgInVMArgs` would simply call `findArgInVMArgsImpl` and pass in false for `doConsumeIgnoredArgs`